src/modules.js

```diff
@@ -489,11 +489,10 @@ function exportRuntime() {
       }
     }
   }
-  return runtimeElements.map(function(name) {
-    return maybeExport(name);
-  }).join('\n') + runtimeNumbers.map(function(name) {
-    return maybeExportNumber(name);
-  }).join('\n');
+  var exports = runtimeElements.map(function(name) { return maybeExport(name); });
+  exports = exports.concat(runtimeNumbers.map(function(name) { return maybeExportNumber(name); }));
+  exports = exports.filter(function(name) { return name != '' });
```

Comment:
when can a name be the empty string? (this looks like new code here)

Reply:
This list consists of strings returned from maybeExport, which returns empty string if a name is not exported.. previously this means we were always generating a whole tonne of newlines for now reason.